gapic/templates/%namespace/%name_%version/%sub/services/%service/client.py.j2

```diff
@@ -23,6 +23,37 @@ from .transports.base import {{ service.name }}Transport
 from .transports.grpc import {{ service.name }}GrpcTransport
 
 
+def _get_default_mtls_endpoint(api_endpoint):
```

Comment:
What do you think about making this a static method of the client class?

Reply:
made it a static method